lib/state/action-types.ts

```diff
@@ -101,15 +102,18 @@ export type SetUnsyncedNoteIds = Action<
   { noteIds: T.EntityId[] }
 >;
 export type ShowAllNotes = Action<'SHOW_ALL_NOTES'>;
-export type ShowUntaggedNotes = Action<'SHOW_UNTAGGED_NOTES'>;
```

Comment:
Is this going to cause regression and remove the untagged notes option which was added?

Reply:
Nevermind. I see added back. I didn't go far enough.